web-src/src/components/GenerateButton.js

```diff
@@ -60,7 +61,9 @@ export function GenerateButton() {
         parameters,
         temperature,
       }]);
-      await saveResults();
+      if (runMode !== RUN_MODE_CF) {
```

Comment:
`if (runMode !== aemService.isContentFragmentsRunMode)`

Reply:
Let's tackle this in a follow-up PR.